master/getting-started/kubernetes/upgrade/setup.md

```diff
@@ -59,7 +59,7 @@ Docs for this are coming soon....
 You must configure `calico-upgrade` so that it can connect to both of the
 following:
 
-- [The existing etcdv2 datastore used by Calico v2.6.4](#configuring-calico-upgrade-to-connect-to-the-etcdv2-datastore)
+- [The existing etcdv2 datastore used by Calico v2.6.4 or later](#configuring-calico-upgrade-to-connect-to-the-etcdv2-datastore)
 
 - [The etcdv3 server you plan to use for Calico v3.0.0](#configuring-calico-upgrade-to-connect-to-the-etcdv3-server)
```

Comment:
I'd replace server -> cluster (since it can often be a cluster of servers rather than a single server)